src/meds_torch/models/eic_forecasting.py

```diff
@@ -162,31 +223,163 @@ def compute(self):
         results = {
             "auroc": self.auroc.compute(),
         }
-        results["top_1_accuracy"] = self.top_1_accuracy.compute()
-        results["top_5_accuracy"] = self.top_5_accuracy.compute()
-        results["top_10_accuracy"] = self.top_10_accuracy.compute()
+        results.update(self.accuracy_metrics.compute())
```

Comment:
Just put this in the main dictionary like AUROC too -- no reason to use two dictionary syntaxes.